Dockerfile

```diff
@@ -1,23 +1,16 @@
-ARG BASE_IMAGE=docker.io/fedora:32
+FROM quay.io/ansible/python-builder:latest as builder
+# =============================================================================
 
-FROM ${BASE_IMAGE}
```

Comment:
@shanemcd do we need to support customizing the base image? There was some very important reason we did this...

Reply:
I _think_ I just added this back when I was unsure what it would be, and got tired of updating the file. We can iterate more on this in the future if we need to, but I think for now this is fine.